src/accessControl/acp/index.js

```diff
@@ -221,14 +262,56 @@ export default class AcpAccessControlStrategy {
 
   #policyUrl;
 
+  #policiesContainer;
+
   #fetch;
 
   constructor(originalWithAcr, policiesContainer, fetch) {
     this.#originalWithAcr = originalWithAcr;
+    this.#policiesContainer = policiesContainer;
```

Comment:
This will break with some recent changes I've done in https://github.com/inrupt/pod-browser/pull/243, where I'm passing the string `policiesContainerUrl` instead of the ResourceInfo `policiesContainer`. This is because when you're visiting someone else's Pod, you probably won't be able to load the container itself, but you should be able to know it URL.